ajax/libs/jqcloud/package.json

```diff
@@ -11,5 +11,17 @@
   "repository": {
     "type": "git",
     "url": "https://github.com/lucaong/jQCloud"
+  },
+  "autoupdate": {
+    "source": "git",
+    "target": "git://github.com/lucaong/jQCloud.git",
+    "fileMap": [
+      {
+        "basePath": "jqcloud",
+        "files": [
+          "**/*"
```

Comment:
If we use `**/*`, how do we avoid adding filename with version number for future versions?